### DISCUSSION TITLE
NEW docker image tcc-public

### DISCUSSION THREAD

**Discussion by Ho-Tung** [IvanaXu/TianChiProj#11] [Q&A] [answered]
> registry.cn-shanghai.aliyuncs.com/tcc-public/python:3
this registry in docker python env python is under the 3.10, how i update it ???

**Comment by IvanaXu** [comment, marked as answer]
> https://tianchi.aliyun.com/forum/postDetail?postId=67720

* registry.cn-shanghai.aliyuncs.com/tcc-public/python:3
U can change it to 
* registry.cn-shanghai.aliyuncs.com/tcc_public/python:3.10
